tests/data/test_converter_public_trades.py

```diff
@@ -0,0 +1,470 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from freqtrade.constants import DEFAULT_TRADES_COLUMNS
+from freqtrade.data.converter import populate_dataframe_with_trades
+from freqtrade.data.converter.orderflow import trades_to_volumeprofile_with_total_delta_bid_ask
+from freqtrade.data.converter.trade_converter import trades_list_to_df
+
+
+BIN_SIZE_SCALE = 0.5
+
+
+def read_csv(filename, converter_columns: list = ["side", "type"]):
+    return pd.read_csv(
+        filename,
+        skipinitialspace=True,
+        infer_datetime_format=True,
+        index_col=0,
+        parse_dates=True,
+        converters={col: str.strip for col in converter_columns},
+    )
+
+
+@pytest.fixture
+def populate_dataframe_with_trades_dataframe(testdatadir):
+    return pd.read_feather(testdatadir / "orderflow/populate_dataframe_with_trades_DF.feather")
+
+
+@pytest.fixture
+def populate_dataframe_with_trades_trades(testdatadir):
+    return pd.read_feather(testdatadir / "orderflow/populate_dataframe_with_trades_TRADES.feather")
+
+
+@pytest.fixture
+def candles(testdatadir):
+    return pd.read_json(testdatadir / "orderflow/candles.json").copy()
+
+
+@pytest.fixture
+def public_trades_list(testdatadir):
+    return read_csv(testdatadir / "orderflow/public_trades_list.csv").copy()
+
+
+@pytest.fixture
+def public_trades_list_simple(testdatadir):
+    return read_csv(testdatadir / "orderflow/public_trades_list_simple_example.csv").copy()
+
+
+@pytest.fixture
```

Comment:
If we assume to keep this (read all other comments first!) - it should be `autouse=True`.
that avoids having a unused fixture in subsequent tests, which might be forgotten to be added to new tests (or worse, removed assuming it's unused).